PostProcess/ScreenSpaceReflection/src/ScreenSpaceReflection.cpp

```diff
@@ -75,121 +74,39 @@ static DILIGENT_CONSTEXPR DepthStencilStateDesc DSS_StencilReadComparisonEqual{
 namespace
 {
 
-RefCntAutoPtr<IShader> CreateShaderFromFile(IRenderDevice*          pDevice,
-                                            IRenderStateCache*      pStateCache,
-                                            const Char*             FileName,
-                                            const Char*             EntryPoint,
-                                            SHADER_TYPE             Type,
-                                            const ShaderMacroArray& Macros = {})
-{
-    ShaderCreateInfo ShaderCI;
-    ShaderCI.EntryPoint                      = EntryPoint;
-    ShaderCI.FilePath                        = FileName;
-    ShaderCI.Macros                          = Macros;
-    ShaderCI.SourceLanguage                  = SHADER_SOURCE_LANGUAGE_HLSL;
-    ShaderCI.Desc.ShaderType                 = Type;
-    ShaderCI.Desc.Name                       = EntryPoint;
-    ShaderCI.pShaderSourceStreamFactory      = &DiligentFXShaderSourceStreamFactory::GetInstance();
-    ShaderCI.Desc.UseCombinedTextureSamplers = true;
-    return RenderDeviceWithCache<false>{pDevice, pStateCache}.CreateShader(ShaderCI);
-}
-
-RefCntAutoPtr<IShader> CreateShaderFromSource(IRenderDevice*          pDevice,
-                                              IRenderStateCache*      pStateCache,
-                                              const Char*             Source,
-                                              const Char*             EntryPoint,
-                                              SHADER_TYPE             Type,
-                                              const ShaderMacroArray& Macros = {})
-{
-    ShaderCreateInfo ShaderCI;
-    ShaderCI.EntryPoint                      = EntryPoint;
-    ShaderCI.Source                          = Source;
-    ShaderCI.Macros                          = Macros;
-    ShaderCI.SourceLanguage                  = SHADER_SOURCE_LANGUAGE_HLSL;
-    ShaderCI.Desc.ShaderType                 = Type;
-    ShaderCI.Desc.Name                       = EntryPoint;
-    ShaderCI.pShaderSourceStreamFactory      = &DiligentFXShaderSourceStreamFactory::GetInstance();
-    ShaderCI.Desc.UseCombinedTextureSamplers = true;
-    return RenderDeviceWithCache<false>{pDevice, pStateCache}.CreateShader(ShaderCI);
-}
-
 ITextureView* GetInternalResourceSRV(const RefCntAutoPtr<IDeviceObject>& pDeviceObject)
```

Comment:
These functions are also defined in the context - it is worth making them public to avoid duplication

Reply:
These functions have been moved to `InternalResourceExtractor.hpp`